index.Rmd

```diff
@@ -56,11 +56,11 @@ teaching.
 
 Educators newly adopting Jupyter can be overwhelmed by having to navigate the
 ecosystem of tools and content. They could study many examples, or consume a
-myriad of blog posts and talk videos to distill the patterns of good practices
+myriad of blog posts and videos of talks to distill the patterns of good practices
 and technical solutions to serve their students best. Several early adopters,
 having much experience to share, decided to begin collecting this know-how, and
-sharing open documentation about using Jupyter for teaching and learning. The
-result is this open book: a live document of collective experience with Jupyter
+share open documentation about using Jupyter for teaching and learning. The
+result is this open book: a live document of collective experiences with Jupyter
```

Comment:
minor suggestion here: "a living document that captures the experiences of community members using Jupyter in education". Thoughts?

Reply:
Sounds good